tests/app/questionnaire/conftest.py

```diff
@@ -831,5 +1022,33 @@ def mock_schema():
 
 
 @pytest.fixture
-def placeholder_transform():
-    return PlaceholderTransforms(language="en")
+def placeholder_transform(schema_placeholder_renderer):
+    schema, renderer = schema_placeholder_renderer
+    return PlaceholderTransforms(language="en", schema=schema, renderer=renderer)
+
+
+@pytest.fixture
+def schema_placeholder_renderer(section_with_labels):
+    schema = QuestionnaireSchema(section_with_labels)
+    answer_id = AnswerStore(
+        [
+            {"answer_id": "mandatory-radio-label-answer-1", "value": "Head"},
+            {"answer_id": "mandatory-radio-label-answer-2", "value": "label"},
+        ]
+    )
+    renderer = PlaceholderRenderer(
+        language="en",
+        answer_store=answer_id,
+        list_store=ListStore(),
+        metadata=ImmutableDict({}),
+        response_metadata={},
+        schema=schema,
+    )
+    return schema, renderer
+
+
+@pytest.fixture
+def operation_helper(schema_placeholder_renderer):
+    schema, renderer = schema_placeholder_renderer
+    ops_helper = OperationHelper(language="en", schema=schema, renderer=renderer)
+    return ops_helper
```

Comment:
If we are using this in only one test file, we should move it there, as it makes the test more readable.

Reply:
I prefer `fixtures` in `conftest` to keep separation between `tests` and `fixtures`.  there are couple of tests using this.